entity-management-web/src/main/java/eu/europeana/entitymanagement/web/EMController.java

```diff
@@ -850,25 +875,26 @@ private ResponseEntity<String> checkExistingEntity(
 
   private ResponseEntity<String> redirectDeprecated(
```

Comment:
should be refactored to have only one redirection that serves multiple situations (old europeana ID, deprecation, etc.). The service must identify the redirection URL.